src/components/ActivityFeed/PostHeader.tsx

```
@@ -0,0 +1,380 @@
+
+	useEffect(() => {
+		if (!window || post?.track_no === null) return;
+		let trackDetails = getQueryToTrack(router.pathname.split('/')[1], network);
+		if (!trackDetails) {
+			trackDetails = getTrackData(network, '', post?.track_no);
+		}
+		if (!post?.created_at || !trackDetails) return;
+
+		const prepare = getPeriodData(network, dayjs(post?.created_at), trackDetails, 'preparePeriod');
+
+		const decisionPeriodStartsAt = decidingStatusBlock && decidingStatusBlock.timestamp ? dayjs(decidingStatusBlock.timestamp) : prepare.periodEndsAt;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)